sigmastate/src/test/scala/sigmastate/utxo/BasicOpsSpecification.scala

```diff
@@ -122,6 +134,55 @@ class BasicOpsSpecification extends SigmaTestingCommons
     flexVerifier.verify(verifyEnv, tree, ctxExt, pr.proof, fakeMessage).get._1 shouldBe true
   }
 
+  property("Unit register") {
+    VersionContext.withVersions(activatedVersionInTests, ergoTreeVersionInTests) {
+      if (VersionContext.current.isJitActivated) {
+        test("R1", env, ext,
+          script = "", /* means cannot be compiled
+                         the corresponding script is { SELF.R4[Unit].isDefined } */
+          ExtractRegisterAs[SUnit.type](Self, reg1)(SUnit).isDefined.toSigmaProp,
+          additionalRegistersOpt = Some(Map(
+            reg1 -> UnitConstant.instance
+          ))
+        )
+
+        // TODO frontend: implement missing Unit support in compiler
```

Comment:
For many reasons, better to have a Github issue for such a task

Reply:
created issue https://github.com/ScorexFoundation/sigmastate-interpreter/issues/820